src/i18n/ja.yml

```diff
@@ -0,0 +1,135 @@
+i18n:
+  lang:
+    code: ja
+    display: 日本語
+    ja: true
+  nav:
+    home: ホーム
+    wallets: ウォレット
+    quickstart: クイックスタート
+    faqs: よくある質問
+  footer:
+    copyright: Copyright &copy; 2017-2019 CashShuffle. All rights reserved.
```

Comment:
NIT: Shouldn't this be translated?

Reply:
Translator says at least for Japanese it's better to leave this in English. He intentionally changed it back from the machine translation.